Travis build fails on node 0.12

See https://travis-ci.org/baconjs/bacon.js/jobs/241915957

Here's the relevant part:

```
Updating nvm
nvm.install
4.78s$ nvm install 0.12
Downloading and installing node v0.12.18...
Downloading https://nodejs.org/dist/v0.12.18/node-v0.12.18-linux-x64.tar.xz...
######################################################################## 100.0%
Computing checksum with sha256sum
Checksums matched!
Now using node v0.12.18 (npm v2.15.11)
$ node --version
v0.12.18
$ npm --version
2.15.11
$ nvm --version
0.33.2
before_install.1
16.91s$ npm install -g npm@'>=2.11'
/home/travis/.nvm/versions/node/v0.12.18/bin/npm -> /home/travis/.nvm/versions/node/v0.12.18/lib/node_modules/npm/bin/npm-cli.js
npm@5.0.3 /home/travis/.nvm/versions/node/v0.12.18/lib/node_modules/npm
before_install.2
0.60s$ npm conf set strict-ssl false
npm WARN npm npm does not support Node.js v0.12.18
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm You can find the latest version at https://nodejs.org/
4.74s$ npm install 
npm WARN npm npm does not support Node.js v0.12.18
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm You can find the latest version at https://nodejs.org/
npm ERR! /home/travis/.nvm/versions/node/v0.12.18/lib/node_modules/npm/lib/install.js:298
npm ERR!         [this, (next) => { computeMetadata(this.idealTree); next() }],
npm ERR!                       ^^
npm ERR! Unexpected token =>
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -5,9 +5,6 @@ node_js:
   - 5
   - 6
   - 8
-before_install:
-  - npm install -g npm@'>=2.11'
-  - npm conf set strict-ssl false
 notifications:
   flowdock: 3853ef33f6943539cbbed913648575beg
 script:

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,10 +1,10 @@
 language: node_js
 sudo: false
 node_js:
-  - '0.12'
   - 4
   - 5
   - 6
+  - 8
 before_install:
   - npm install -g npm@'>=2.11'
   - npm conf set strict-ssl false

